target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -82,6 +82,13 @@ broadway)
 	set_usb_led "$board:red:diskmounted"
 	set_wifi_led "$board:red:wps_active"
 	;;
+c20)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
```

Comment:
Drop this line, the LED is handled by the diag.sh script and should be on after boot by default. If the LED is off after boot, the GPIO_ACTIVE value for the LED is wrong in the device tree source file.

Reply:
Thanks. I will check it.